src/main/java/dao/AbstractDao.java

```diff
@@ -0,0 +1,16 @@
+package dao;
```

Comment:
I think it's better to have all your packages such as dao, entity, service, etc in a parent package.